### DISCUSSION TITLE
Nice Job

### DISCUSSION THREAD

**Discussion by sm-tradeboox** [Sithuk/ubuntu-server-zfsbootmenu#24] [General]
Hi,

No issues, just wanted to give you kudos for the work you've done. !!

I have needed to modify your script substantially because my requirements are  simple, but let me know if you want feedback from this noob.

Sam

**Comment by sm-tradeboox** [comment]
Just compliment, not issue :)

**Comment by Sithuk** [comment]
Feedback always welcome :)
The best place for feedback is the reddit thread.
https://www.reddit.com/r/zfs/comments/mj4nfa/ubuntu_server_2104_native_encrypted_root_on_zfs/
